cmd/cloud/deployment.go

```diff
@@ -756,20 +756,12 @@ func deploymentUpdate(cmd *cobra.Command, args []string, out io.Writer) error {
 		return errors.New("Invalid --development-mode value")
 	}
 	if cicdEnforcement != "" && !(cicdEnforcement == enable || cicdEnforcement == disable) {
-		return errors.New("Invalid --cicd-enforcment value")
+		return errors.New("Invalid --cicd-enforcement value")
 	}
 	if cmd.Flags().Changed("enforce-cicd") {
-		if deploymentUpdateEnforceCD && cicdEnforcement == disable {
-			return errors.New("flags --enforce-cicd and --cicd-enforcment contradict each other. Use only --cicd-enforcment")
-		}
-		if !deploymentUpdateEnforceCD && cicdEnforcement == enable {
-			return errors.New("flags --enforce-cicd and --cicd-enforcment contradict each other. Use only --cicd-enforcment")
-		}
-		if deploymentUpdateEnforceCD {
-			cicdEnforcement = enable
-		}
-		if !deploymentUpdateEnforceCD {
-			cicdEnforcement = disable
+		shouldReturn, err1 := validateCICD()
+		if shouldReturn {
+			return err1
```

Comment:
I don't think we need that boolean. The boolean equals `err != nil`.
```suggestion
		if err := validateCICD(); err != nil {
			return err
```